Templates/AzureDevOpsPipeline/README.md

```diff
@@ -1,33 +1,56 @@
   * to run the Wazi Deploy [evidence command](../Common-Backend-Scripts/README.md#49---wazideploy-evidencesh) to generate deployment report and updating the evidence.
   * to [prepare](../Common-Backend-Scripts/README.md#49---preparelogssh) the deployment log files and publish them to the Azure build result.
+* `Deployment` to controlled test environments via the [release pipeline](https://ibm.github.io/z-devops-acceleration-program/docs/branching-model-supporting-pipeline#the-release-pipeline-with-build-packaging-and-deploy-stages) that includes:
+  * to create a git tag to flag the release candidate. 
+  * to retrieve the package from the Azure Artifacts binary repository.
+  * to deploy the package with the Wazi Deploy [deploy command](../Common-Backend-Scripts/README.md#48---wazideploy-deploysh) (Python-based)
+  * to run the Wazi Deploy [evidence command](../Common-Backend-Scripts/README.md#49---wazideploy-evidencesh) to generate deployment report and updating the evidence.
+  * to [prepare](../Common-Backend-Scripts/README.md#49---preparelogssh) the deployment log files and publish them to the Azure build result.
+  * to create a git tag of the commit that was deployed to the production environment.
+* `Cleanup` stage: 
+  * to [delete the build workspace](../Common-Backend-Scripts/README.md#411---deleteworkspacesh) on z/OS Unix System Services.
+
+Depending on your selected deployment technology, review the definitions and (de-)/activate the appropriate steps.
+
+The pipeline uses the Azure concepts `Stage`, `Jobs` and `Tasks`, as well leverages Gitlab templates.
+
+![Azure Release pipeline](images/ado_releasePipeline.png)
+
+## Prerequisites
+
+To leverages this template, access to an Azure DevOps environment and an Azure Pipeline agent that can connect to your mainframe environment is required. Please review the setup instructions of this [techdoc](https://www.ibm.com/support/pages/node/6422813).
 
+The template leverages the [SSH Task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/ssh-v0?view=azure-pipelines) to invoke the [Common Backend scripts](../Common-Backend-Scripts/) via the configured ssh endpoint. Alternatively, the template can be modified to use the [CmdLine task](https://learn.microsoft.com/en-us/azure/devops/pipelines/tasks/reference/cmd-line-v2?view=azure-pipelines) to leverage an alternative communication technology such as Zowe CLI.
 
-The pipeline uses the Azure concepts `Stage`, `Jobs` and `Tasks`:
+The backend scripts need to be configured for the selected technologies to operate correctly.
 
-![](images/ado_pipelineOverview.png)
+The implemented tagging of important commits in the git history, leverages the Azure CLI, that needs to be available on the Azure runner. Please follow the [documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) to make the Azure CLI available to all agents within the agent pool that the pipelines will use.
```

Comment:
Could we rephrase to something like this?
To tag important commits in the history of the application's Git repository, the Azure CLI is used and must be available on the Azure runner. Please follow the [documentation](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli) to install the Azure CLI on all the agents within the agent pool that the pipelines will use.